### PR TITLE
Introducing: the OR LBR8 "Hornet"

### PR DESCRIPTION
**Pull request by maikilangiolo**
## About The Pull Request
A bulky revolver based on the ancient skydriver frame. Though it cannot match the power of the Skyriver, oberth scientists decided that cramming .20 rifle shells and a longer barrel was good enough of an attempt, and they were not wrong.
While the Miller is a handcannon, the consul a solid sidekick, the havelock a pocket gun, and the deckard an ~~anti-replicant~~ anti-FBP revolver, the Hornet sports target sights, a spacious chamber, and good penetration profile.
imageimage

The hornet will be a marshall uplink only gun, who lose their miller in exchange. 
Added a short scope to the skydriver, finally gave it faster bullets, and a tiny bit of recoil because I think it should kick a bit as a tradeoff (should still be minimal, might be adjusted)

## Why It's Good For The Game
new gun with soli niche, baron sprite good, marshall gets some more love with unique weapons instead of being a copy paste. Pretty sure Firefox approved it, but if the PR gets closed, well, so be it.

## Changelog
:cl:
add: added new marshall .20 revolver
del: removed duplicate ammo var in skydriver
tweak: tweaked skydriver: 0.2 more recoil, 30% faster bullets, short scope
/:cl:

### CODE REVIEW

**Review by Firefox13 — COMMENTED** [review]
Code looks okay and so do stats, but people might want to discuss them.
Make sure to notify Kegdo for #7569 edit (holsters holding it)

**Comment by Trilbyspaceclone** [comment]
Just a suggestion for it to stand out more
Rather then it being high damage maybe high pain multiplier?
To be more in like with taking in people alive to collect the bounty on them or something.

**Comment by maikilangiolo** [comment]
> Just a suggestion for it to stand out more Rather then it being high damage maybe high pain multiplier? To be more in like with taking in people alive to collect the bounty on them or something.

It's not high damage. I know there's a 2x dmg multiplier there, but it's halved since it's .20. The theoretical damage is 60, but it's actually 30, 10 less than the consul and 20 less than the miller/mateba, but with 10 more than the miller against heavier armor (revolvers do 40-50 damage a shot).

**Comment by ErisMeleeEnjoyer** [comment]
god it looks sexy as fuck

**Comment by TheShown911** [comment]
Wasn't there a gun ban or something?

**Comment by maikilangiolo** [comment]
Firefox said "OK", and you can see his opinion in the first comment of this PR

**Comment by maikilangiolo** [comment]
ok added the speedloders for it bobob made (best spriter)

File: code/modules/projectiles/guns/projectile/revolver/hornet.dm [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/obj/item/gun/projectile/revolver/hornet
+	name = "OR REV .20 \"LBR-8 Hornet\""
+	desc = "An attempt by Oberth to replicate lost syndicate tech. In order to achieve satisfactory ballistic \
+			perfomance, it sports an usually long barrel and overpressurized chamber. Uses .20 rifle rounds."
```
Comment (hyperioo) [inline code/modules/projectiles/guns/projectile/revolver/hornet.dm:4]:
```suggestion
			performance, it sports an usually long barrel and overpressurized chamber. Uses .20 rifle rounds."
```

**Review by hyperioo — APPROVED** [review]

**Pull request merged**